src/test/resources/application.conf

```diff
@@ -25,6 +25,18 @@ local {
       port = 10050
       productionRoute = "/pillar-two/"
     }
+    businessEQ {
```

Comment:
don't need to do the configuration for each page. This can be done using the url variable in page class.

Reply:
Updated